mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -243,5 +247,11 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 		ARG 2 monitor
 	METHOD method_27466 initFont (Z)V
 		ARG 1 forcesUnicode
+	METHOD method_29042 shouldHideMessage (Ljava/util/UUID;)Z
```

Comment:
What type of messages does this hide?

Reply:
Chat messages, and check if online chat option is enabled

Reply:
would prefer `shouldHideMessage` or `isChatMuted`

Reply:
The parameter is already `sender`, so you know if a sender is muted. also this exists for all game messages than just user messages

On another note, this check is used by narrator as well. Checking `shouldHide` in narrator doesn't make as much sense as checking `isMuted`